examples/java/pom.xml

```diff
@@ -39,58 +39,24 @@
   </properties>
 
   <profiles>
-    <!--
-      A default profile that includes optional dependencies
-      on all of our runners. This is aimed at making it very
-      easy for users to run examples with any runner without
-      any configuration. It is not needed or desirable when
-      testing the examples with a particular runner.
-
-      This profile can be disabled on the command line
-      by specifying -P !include-runners.
-
-      This profile cannot be lifted to examples-parent because
-      it would be automatically deactivated when the Java 8
-      profile were activated.
-    -->
+    <!-- Include SparkRunner and deps with -P spark-runner -->
     <profile>
-      <id>include-runners</id>
-      <activation>
-        <activeByDefault>true</activeByDefault>
-      </activation>
+      <id>spark-runner</id>
       <dependencies>
         <dependency>
           <groupId>org.apache.beam</groupId>
-          <artifactId>beam-runners-direct-java</artifactId>
-          <version>${project.version}</version>
-          <scope>runtime</scope>
-          <optional>true</optional>
-        </dependency>
-
-        <dependency>
-          <groupId>org.apache.beam</groupId>
-          <artifactId>beam-runners-flink_2.10</artifactId>
-          <version>${project.version}</version>
-          <scope>runtime</scope>
-          <optional>true</optional>
-        </dependency>
-
-        <dependency>
-          <groupId>org.apache.beam</groupId>
-          <artifactId>beam-runners-google-cloud-dataflow-java</artifactId>
+          <artifactId>beam-runners-spark</artifactId>
           <version>${project.version}</version>
           <scope>runtime</scope>
           <optional>true</optional>
```

Comment:
Copy pasta. Previously, we had integration test modules that depended on examples and needed to not transitively bring in e.g. Spark dependencies. Now that we have fine-grained profiles I can remove.